src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" 
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./style.css">
+    <meta charset="UTF-8">
+    <meta 
+      name="viewport"
+      content="
+        width=device-width, 
+        user-scalable=no, 
+        initial-scale=1.0,
+        maximum-scale=1.0, 
+        minimum-scale=1.0
+    ">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        class="card__picture" 
+        src="./images/imac.jpeg" 
+        alt="apple a1419 imac 27"
+      >
+
+      <h2 class="card__main-text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__product-code">
+        Product code: 195434
+      </div>
+
+      <div class="card__star-reviews">
+        <ul class="star star--4">
+          <li class="star__stars"></li>
```

Comment:
```suggestion
        <ul class="stars stars--4">
          <li class="stars__star"></li>
```
![image](https://user-images.githubusercontent.com/96409563/224533937-797ebcea-fd0b-47fe-89f9-1f8842c8c903.png)